src/imex.rs

```diff
@@ -631,6 +804,129 @@ async fn export_backup_inner(
     Ok(())
 }
 
+async fn export_backup_iroh(
+    context: &Context,
+    dir: &Path,
+    passphrase: String,
+) -> Result<iroh_share::SenderTransfer> {
+    // get a fine backup file name (the name includes the date so that multiple backup instances are possible)
+    let now = time();
+    let (temp_db_path, temp_path, dest_path) = get_next_backup_path(dir, now)?;
+    let sender_db_path = dir.join("iroh_db");
+    let _d1 = DeleteOnDrop(temp_db_path.clone());
+    let _d2 = DeleteOnDrop(temp_path.clone());
+    let _d3 = DeleteOnDrop(sender_db_path.clone());
+
+    context
+        .sql
+        .set_raw_config_int("backup_time", now as i32)
+        .await?;
+    sql::housekeeping(context).await.ok_or_log(context);
+
+    context
+        .sql
+        .execute("VACUUM;", paramsv![])
+        .await
+        .map_err(|e| warn!(context, "Vacuum failed, exporting anyway {}", e))
+        .ok();
+
+    ensure!(
+        context.scheduler.read().await.is_none(),
+        "cannot export backup, IO is running"
+    );
+
+    info!(
+        context,
+        "Backup '{}' to '{}'.",
+        context.get_dbfile().display(),
+        dest_path.display(),
+    );
+
+    context
+        .sql
+        .export(&temp_db_path, passphrase)
+        .await
+        .with_context(|| format!("failed to backup plaintext database to {:?}", temp_db_path))?;
+
+    let res = export_backup_iroh_inner(context, &temp_db_path, &temp_path).await;
+
+    match res {
+        Ok(dir_builder) => {
+            let port = 9990;
```

Comment:
similar concern about hardcoding the port

Reply:
fixed